Source/WebInspectorUI/UserInterface/Views/NodeOverlayListSection.js

```diff
@@ -125,11 +125,6 @@ WI.NodeOverlayListSection = class NodeOverlayListSection extends WI.View
             itemContainerElement.append(swatch.element);
 
             swatch.addEventListener(WI.InlineSwatch.Event.ValueChanged, (event) => {
-                if (checkboxElement?.checked)
-                    domNode.showLayoutOverlay({color: event.data.value});
-            }, swatch);
-
-            swatch.addEventListener(WI.InlineSwatch.Event.Deactivated, (event) => {
```

Comment:
I think the thinking behind this was that then we wouldn't be writing to `localStorage` a bazillion times if the developer drags around inside the `WI.ColorSquare`.  Perhaps we could/should use a `Debouncer`?

I guess it maybe can't really be that much worse than having a protocol command be dispatched with every change (which is what was/is happening), so maybe not that big of a deal.

Reply:
I think the complexity of caching the latest color value to then debounce saving that value to `localStorage` (and we should probably debounce sending the protocol command at that point) is probably the wrong tradeoff unless we have an actual performance issue we are addressing.